pipelines/test/fonts.yaml

```diff
@@ -0,0 +1,50 @@
+name: fonts
+
+needs:
+  packages:
+    - file
+    - py3-fonttools
+
+pipeline:
+  - name: font usability test
+    runs: |
+      # Get our font package name
+      font_pkg=$(basename ${{targets.contextdir}})
```

Comment:
${{subpkg.name}} FTW

Reply:
I think there is enough usage of `$(basename ${{targets.contextdir}})` that we should replace these en masse and not hold up this particular PR until this is fixed in it.